openpype/hosts/maya/startup/userSetup.py

```diff
@@ -13,23 +12,40 @@
 
 print("Starting OpenPype usersetup...")
 
+settings = get_project_settings(os.environ['AVALON_PROJECT'])
+
+# Loading plugins explicitly.
+if settings["maya"]["explicit_plugins_loading"]["enabled"]:
+    def _explicit_load_plugins():
+        project_settings = get_project_settings(os.environ["AVALON_PROJECT"])
+        maya_settings = project_settings["maya"]
+        explicit_plugins_loading = maya_settings["explicit_plugins_loading"]
+        if explicit_plugins_loading["enabled"]:
+            for plugin in explicit_plugins_loading["plugins_to_load"]:
+                if plugin["enabled"]:
+                    print("Loading " + plugin["name"])
+                    try:
+                        cmds.loadPlugin(plugin["name"], quiet=True)
+                    except RuntimeError as e:
+                        print(e)
+
+    cmds.evalDeferred(
+        _explicit_load_plugins,
+        lowestPriority=True
+    )
```

Comment:
Should there be a comment above this about why we're deferring the loading of the plug-ins? Why couldn't we load directly?

Reply:
Yup will do